app/templates/partials/summary/display-answer-label.html

```diff
@@ -0,0 +1 @@
+<div class="summary__label mars" id="{{ answer.id }}-label" data-qa="{{ answer.id }}-label">{{ answer.label|striptags }}</div>
```

Comment:
Do we need a `data-qa` and `id` when they are both the same? One or the other but both seems overkill.

Reply:
I was copying display-answer.html.  Is that one wrong also?  If so then I can make a trello card to remove one or the other if you want.

Reply:
Keep it then if used elsewhere. Seems excessive rather than wrong.